tests/run-tests.sh

```diff
@@ -20,5 +20,23 @@ if [[ "${TEST_TYPE}" == "aio" ]]; then
                                                      -e ansible_connection=local \
                                                      ${TEST_DIR}/tests/playbooks/test.yml
 
-    source ${TEST_DIR}/scripts/deploy-all.sh
+    # Defaults missing environment variable with "all"
+    if [ -z "${RE_JOB_SCENARIO}" ]; then
```

Comment:
At the top of the script this could be defined as an export

``` shell
export RE_JOB_SCENARIO="${RE_JOB_SCENARIO:-all}"
```

Reply:
Good call